thirdparty/README.md

```diff
@@ -152,6 +152,19 @@ Files extracted from the upstream source:
 
 TODO.
 
+## openvr
+
+- Upstream: https://github.com/ValveSoftware/openvr
+- Version: 1.0.6
+- License: Valve (looks like MIT)
```

Comment:
GitHub seems to detect `BSD 3-clause "New" or "Revised" License`, see https://github.com/ValveSoftware/openvr/blob/master/LICENSE